test/test_library/functional/test_library.py

```diff
@@ -147,18 +147,19 @@ def test_database_query(self):
         for ref, vid in tests.items():
             ip, results = r.resolve_ref_all(ref)
 
-            self.assertEqual(1, len(results))
+            self.assertEquals(1, len(results))
 
             first = results.values().pop(0)
             vid2 = first.vid if not first.partitions else first.partitions.values()[0].vid
 
             self.assertEquals(vid, vid2)
 
+    @unittest.skip(
```

Comment:
That test tests test/bundles/testbundle installation. If testbundle has 43 columns it should test for 43 columns added to db.

TODO: replace 45 with 43.